tests/test_lab_test.py

```diff
@@ -0,0 +1,120 @@
+# SPDX-License-Identifier: Apache-2.0
+# pylint: disable=duplicate-code
+
+# Standard
+from unittest.mock import patch
+import os
+import platform
+import sys
+
+# Third Party
+from click.testing import CliRunner
+
+# First Party
+from instructlab import lab
+
+INPUT_DIR = "test_generated"
+TRAINING_RESULTS_DIR = "training_results"
+CHECKPOINT_DIR_NAME = "checkpoint-1"
+MERGED_MODEL_DIR_NAME = "merged_model"
+FINAL_RESULTS_DIR_NAME = "final"
+CHECKPOINT_DIR = TRAINING_RESULTS_DIR + "/" + CHECKPOINT_DIR_NAME
+MERGED_MODEL_DIR = TRAINING_RESULTS_DIR + "/" + MERGED_MODEL_DIR_NAME
+FINAL_RESULTS_DIR = TRAINING_RESULTS_DIR + "/" + FINAL_RESULTS_DIR_NAME
+LINUX_GGUF_FILE = FINAL_RESULTS_DIR + "/ggml-model-f16.gguf"
+MODEL_DIR = "model"
+ENCODING = "UTF-8"
+
+
+def setup_input_dir():
```

Comment:
How do you purge all of that when the test ends?

Reply:
@leseb I just copy it from existing `test_lab_train.py`. It doesn't do clean up after tests. Can this be done in a separate PR?

Reply:
Sure, sorry I missed this was already there. Followup is ok!